<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrowClock</class>
 <widget class="QMainWindow" name="ArrowClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArrowClock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cb_set_time">
        <property name="text">
         <string>Установить время</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbl_h">
          <property name="text">
           <string>Введите часы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_hours">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_m">
          <property name="text">
           <string>Введите минуты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_minutes">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lbl_s">
          <property name="text">
           <string>Введите секунды</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_seconds">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_change_time"/>
    <addaction name="action_dark_theme"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_change_time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Установить время</string>
   </property>
  </action>
  <action name="action_dark_theme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Выбрать тему</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
